src/eval-frame/components/reps/tiny-reps.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import styled from "react-emotion";
+
+const numberColor = "rgb(28, 0, 207)";
+const boolColor = numberColor;
+const nullUndefinedColor = "#808080";
+const classInfoColor = "#808080";
+const stringColor = "rgb(196, 26, 22)";
+const separatorColor = "black";
+const dateColor = stringColor;
+const symbolColor = stringColor;
+const errorColor = "";
+const functionColor = "rgb(28, 30, 207)";
+const ellipsisColor = "#111 ";
+
+const RepBaseStyledSpan = color => styled("span")`
```

Comment:
@hamilton i'm not sure if i'm using styled components idiomatically/optimally/correctly throughout this PR. lmk what you think.